.husky/pre-commit

```diff
@@ -10,6 +10,29 @@ if [ "$current_branch" = "main" ]; then
     fi
 fi
 
-# Allow commit
-exit 0
+# If env var viDUST_LINT_ON_COMMIT = 1, run the linter
+if [ -z "$DUST_LINT_ON_COMMIT" ]; then
+    exit 0
+fi
+
+
+# Get the root path of the git repository
+root_path=$(git rev-parse --show-toplevel)
 
+# List the directories to check for changes
+directories=("front" "connectors")
+
+# For each directory, check if any files have changed
+for directory in "${directories[@]}"; do
+    if git diff --cached --name-only | grep -qE "^$directory/"; then
+        # Run the linter on root_path concatenated with the directory
+        if ! npm run lint --prefix $root_path/$directory; then
```

Comment:
Usually people go with https://www.npmjs.com/package/lint-staged that pretty output linter errors.

Reply:
Thanks for the pointer, i'll keep the PR as it to avoid distracting myself too much :)